evap/staff/tests/test_views.py

```diff
@@ -489,8 +490,8 @@ def test_warning_handling(self):
         self.assertContains(
             reply,
             "The existing user would be overwritten with the following data:<br />"
-            " -  None None, lucilia.manilium@institution.example.com (existing)<br />"
-            " -  Lucilia Manilium, lucilia.manilium@institution.example.com (new)",
+            " -  None None, lucilia.manilium@institution.example.com (existing) [{}]<br />"
+            " -  Lucilia Manilium, lucilia.manilium@institution.example.com (new)".format(user_edit_link(user.pk)),
```

Comment:
Could you please add a unit test for user_edit_link? Currently, all tests just test that method against itself, so we wouldn't notice if it was broken.